website_event_share/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+1. Go to the website events menu.
+2. Open one of your events.
+3. Click on the customize button.
+4. Toggle the share button option to enable/disable the share links block.
```

Comment:
In general it's better to use `#.` for bullet lists, to use automatic numeration.